docs/rest_api_guide/03_RESTstatistics.rst

```diff
@@ -0,0 +1,177 @@
+
+.. list-table::
+    :widths: 20 50
+    :header-rows: 1
+
+    * - Key
+      - Description
+    * - interval
+      - The interval in seconds between successive statistics values
+    * - statistics[].BUFFERED
+      - Readings currently in the Fledge buffer
+    * - statistics[].DISCARDED
+      - Readings discarded by the South Service before being  placed in the buffer. This may be due to an error in the readings themselves.
+    * - statistics[].PURGED
+      - Readings removed from the buffer by the purge process
+    * - statistics[].READINGS
+      - Readings received by Fledge
+    * - statistics[].*NORTH_TASK_NAME*
+      - The number of readings sent to the PI system via the OMF plugin with north instance name
+    * - statistics[].UNSENT
+      - Readings filtered out in the send process
+    * - statistics[].UNSNPURGED
+      - Readings that were purged from the buffer before being sent
+    * - statistics[].*ASSET-CODE*
+      - The number of readings received by Fledge since startup with name *asset-code*
+
+
+**Example**
+
+.. code-block:: console
+
+  $ curl -s http://localhost:8081/fledge/statistics/history?limit=2
+  {
+    "interval": 15,
+    "statistics": [
+      {
+        "history_ts": "2020-06-01 11:21:04.357",
+        "READINGS": 0,
+        "BUFFERED": 0,
+        "UNSENT": 0,
+        "PURGED": 0,
+        "UNSNPURGED": 0,
+        "DISCARDED": 0,
+        "Readings Sent": 0
+      },
+      {
+        "history_ts": "2020-06-01 11:20:48.740",
+        "READINGS": 0,
+        "BUFFERED": 0,
+        "UNSENT": 0,
+        "PURGED": 0,
+        "UNSNPURGED": 0,
+        "DISCARDED": 0,
+        "Readings Sent": 0
+      }
+    ]
+  }
+  $
+
+
+GET statistics/rate
+~~~~~~~~~~~~~~~~~~~
+
+``GET /fledge/statistics/rate`` - return a set of rates for a set of statistics. This interface returns the rate of a statistic value in counts per minute over a specified set of averages. It is passed two parameters, a comma separated list of intervals in minutes and a comma separated list of statistics.
+
+**Request Parameters**
+
+  - **statistics** - a comma separated list of statistics values to return
+
+  - **periods** - a comma separated list of time periods in minutes. The corresponding rate that will be returned for a given value X is the counts per minute over the previous X minutes.
+
+**Example**
+
+.. code-block:: console
+
+   $ curl http://localhost:8081/fledge/statistics/rate?statistics=Readdings%20Sent\&periods=1,5,15,30,60
+   {
+      "rates": {
+        "READINGS": {
+          "1": 12.938816958618938,
+          "5": 12.938816958618938,
+          "15": 12.938816958618938,
+          "30": 12.938816958618938,
+          "60": 12.938816958618938
+        },
+        "READINGS SENT": {
```

Comment:
Readings Sent - As you have changed the response without UPPER case, so key should be as per the request